core/src/main/scala/org/apache/spark/scheduler/SparkListenerBus.scala

```diff
@@ -71,7 +71,6 @@ private[spark] trait SparkListenerBus
         listener.onNodeUnblacklisted(nodeUnblacklisted)
       case blockUpdated: SparkListenerBlockUpdated =>
         listener.onBlockUpdated(blockUpdated)
-      case logStart: SparkListenerLogStart => // ignore event log metadata
```

Comment:
I haven't looked at this closely, but will allowing this through cause any issues? Why was it ignored before?